target/linux/ramips/image/mt76x8.mk

```diff
@@ -652,6 +652,19 @@ define Device/tplink_archer-mr200-v5
 endef
 TARGET_DEVICES += tplink_archer-mr200-v5
 
+define Device/tplink_archer-mr200-v6
+  $(Device/tplink-v2-lzma-custom)
```

Comment:
Is this not working with `Device/tplink-v2`?

Reply:
Why do you need `Device/tplink-v2-lzma-custom`? 
If you need `lzma -d22` you can use `Device/tplink-v2` and then overwrite the `KERNEL` line.